src/Paprika/Store/StorageFanOutPage.cs

```diff
@@ -102,6 +102,30 @@ public void Report(IReporter reporter, IPageResolver resolver, int level)
             }
         }
     }
+    public void Destroy(IBatchContext batch, in NibblePath prefix)
+    {
+        // Destroy the Id entry about it
+        Set(prefix, ReadOnlySpan<byte>.Empty, batch);
+
+        var map = new SlottedArray(Data.Data);
+        map.DeleteByPrefix(prefix);
+        var sliced = prefix.SliceFrom(ConsumedNibbles);
```

Comment:
slice can be done only if there's a child, move donw?

Reply:
Done.